Dockerfile

```diff
@@ -1,38 +1,47 @@
-
-
-FROM ubuntu:21.04 as build
+FROM ubuntu:20.04 as build
 
 ENV LANG C.UTF-8
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN dpkg --add-architecture i386 && \
-    apt-get update && \
+RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-        binutils:i386 \
-        gcc-10:i386 \
-        g++-10:i386 \
-        python3.10 \
-        python \
+        binutils \
+        gcc-10 \
+        g++-10 \
+	p7zip-full \
```

Comment:
nitpick but if we're indenting with spaces for the rest of the file this shouldn't be a tab